frontend/catalyst/jit.py

```diff
@@ -374,6 +374,35 @@ def sum_abstracted(arr):
     if fn is None:
         return functools.partial(qjit, **kwargs)
 
+    # !!! TODO: fix jax.scipy numerical failures with properly fetched lapack calls
+    # https://app.shortcut.com/xanaduai/story/70899/find-a-system-to-automatically-create-a-custom-call-library-from-the-one-in-jax
+    # https://github.com/PennyLaneAI/catalyst/issues/753
+    # https://github.com/PennyLaneAI/catalyst/issues/1071
+    if "jax.scipy." in inspect.getsource(fn):
```

Comment:
This is a very bad way to check if the user called `jax.scipy`. Any suggestions are welcome.

Reply:
The problem is we can't change the source of `jax.scipy.linalg.expm` and raise warning over there.

Reply:
no longer relevant since we patch instead of warn now